tests/test_selector.py

```diff
@@ -90,9 +90,12 @@ def test_representation_slice(self):
         body = u"<p><input name='{}' value='\xa9'/></p>".format(50 * 'b')
         sel = self.sscls(text=body)
 
-        representation = "<Selector xpath='//input/@name' data='{}'>".format(40 * 'b')
         if six.PY2:
             representation = "<Selector xpath='//input/@name' data=u'{}'>".format(40 * 'b')
+        elif six.PY34:
+            representation = "<Selector xpath='//input/@name' data='{}'>".format('...')
```

Comment:
Hm, interesting, the output here is being fully discarded by `shorten` because it can't break it up in words.
Maybe we should just use our own shorten version and skip `textwrap.shorten` at all, for better consistency?